tests/system/Database/Builder/UpdateTest.php

```diff
@@ -350,7 +403,7 @@ public function testSetWithoutEscape()
             ->where('id', 2)
             ->update(null, null, null);
 
-        $expectedSQL   = 'UPDATE "mytable" SET field = field+1 WHERE "id" = 2';
+        $expectedSQL   = 'UPDATE "mytable" SET "field" = field+1 WHERE "id" = 2';
```

Comment:
@ytetsuro Does this SQL statement run without errors on Oracle?

Reply:
We must write code for Oracle like this:
```php
$builder
    ->set('field', '"field"+1', false)
    ->where('id', 2)
    ->update(null, null, null);
```
But we still need this PR to quote the `field` immediately after `SET`.

Reply:
> Does this SQL statement run without errors on Oracle?

Thanks, response.

No.
As you can imagine, the `field ` is not escaped, so an error occurs.